websocket.go

```diff
@@ -672,9 +711,27 @@ func (c *wsConn) readFrame(ctx context.Context, r io.Reader) {
 		return
 	}
 
-	c.frameExecQueue <- buf
-	if len(c.frameExecQueue) > cap(c.frameExecQueue)/2 {
-		log.Warnw("frame executor queue is backlogged", "queued", len(c.frameExecQueue), "cap", cap(c.frameExecQueue))
+	var frame frame
+	if err := json.Unmarshal(buf, &frame); err != nil {
```

Comment:
This unmarshal was done in a separate goroutine very deliberately - we want to avoid blocking in the network goroutine at all costs, having it here will cause timeouts with larger RPC requests (e.g. the few-gigabyte one returning all market deals)

Reply:
fair enough, I was just trying out a few things to understand how the flow works